diff --git a/node_modules/@nuxthq/ui/dist/runtime/components/data/Table.vue b/node_modules/@nuxthq/ui/dist/runtime/components/data/Table.vue
index 1c56afc..46851ad 100644
--- a/node_modules/@nuxthq/ui/dist/runtime/components/data/Table.vue
+++ b/node_modules/@nuxthq/ui/dist/runtime/components/data/Table.vue
@@ -178,6 +178,7 @@ export default defineComponent({
       } else {
         sort.value = { column: column.key, direction: column.direction || "asc" };
       }
+      attrs.onSort({'column': column.key, 'direction': column.direction});
     }
     function onSelect(row) {
       if (!attrs.onSelect) {
